tweetledee/userrss_nocache.php

```diff
@@ -112,7 +112,7 @@
 header("Content-Type: application/rss+xml");
 header("Content-type: text/xml; charset=utf-8");
 
-$renderer = new RssRenderer();
+$renderer->using_client($client);
```

Comment:
Isn't the renderer missing here @Riduidel ?

Reply:
yes ideed, and it is even present in [#78](https://github.com/tweetledee/tweetledee/pull/78/files#diff-0613fc4cd74ffe369eeb2bfdeec69290R123)